test/dlc_tests/container_tests/bin/testSmprofiler

```diff
@@ -42,7 +42,8 @@ if [[ $FRAMEWORK == "tensorflow2" ]]; then
     pip install tensorflow-datasets==${TENSORFLOW_DATASETS_VERSION}
 fi
 if [[ $FRAMEWORK == "tensorflow2" ]] || [[ $FRAMEWORK == "pytorch" ]]; then
-    for test_file in tests/profiler/${FRAMEWORK}/*
+    echo "Rajan : ${PWD}............"
```

Comment:
The smprofiler tests also clone the sagemaker-debugger repository at the version installed in the DLC. You may want to change the branch to `master` similar to how you have done it in the smdebug test.